docs/example-scenario/apps/devops-dotnet-webapp.md

```diff
@@ -79,7 +80,7 @@ This [decision tree for Azure compute services](/azure/architecture/guide/techno
 
 ### Prerequisites
 
-- You must have an existing Azure account. If you don't have an Azure subscription, create a [free account][azure-free-account] before you begin.
+- You must have an existing Azure account. If you don't have an Azure subscription, create a [free account](https://azure.microsoft.com/free) before you begin.
```

Comment:
Are we supposed to use the query string to track clicks?

Reply:
Fixed.